source/config/__init__.py

```diff
@@ -1005,7 +1005,7 @@ def __iter__(self):
 				keys.add(key)
 				yield key
 
-	def iteritems(self):
+	def items(self):

```

Comment:
If this is trying to mimic a dictionary, it would be nice if there was documentation on the class to specify that intent. I also wonder if this should instead be inheriting from `UserDict` or `collections.abc.MutableMapping`

No need for you to do anything here, but opinions welcome.

Reply:
I have looked at MutableMapping indeed. It does require us to implement additional methods we don't use, though.